docs/en/openmldb_sql/data_types/string_types.md

```diff
@@ -7,7 +7,7 @@ A string is a sequence of bytes or characters, enclosed within either single quo
 "another string"
 ```
 
-| type   | size | use        |
-| :----- | :--- | :--------- |
-| STRING | 2M   | 变长字符串 |
+| type   | size | use                    |
+| :----- | :--- | :--------------------- |
+| STRING | 2M   | String variable length |
```

Comment:
string variable length => variable-length character strings
refer to the VARCHAR in MySQL